packages/server/api/controllers/teamsController.js

```diff
@@ -49,6 +49,19 @@ const createTeam = async (body) => {
   };
 };
 
+const validateTeamCode = async (body) => {
+  const { teamCode } = body;
+
+  const results = await knex('Teams').where({ team_code: teamCode });
+
+  if (results.length === 0) {
+    throw new Error('Team not found');
```

Comment:
Perhaps would be better to not throw an error, but to send a response with 404 code?

Reply:
![Uploading Screenshot 2024-04-21 at 17.21.32.png…]()

Reply:
I cannot see the pic 
<img width="415" alt="image" src="https://github.com/HackYourFuture-CPH/team26-RetroRealm/assets/64137994/d821d746-03f2-45f3-ba12-c5a3eac30d4c">

Reply:
<img width="1512" alt="Screenshot 2024-04-21 at 17 21 32" src="https://github.com/HackYourFuture-CPH/team26-RetroRealm/assets/104554991/01212a0f-bbd5-4e97-a08b-52d319864570">
we have something like this when user enter the wrong code

Reply:
I saw this, but I mean in terms of the API, it is perhaps not the best to throw an error here, because and error did not happen. 
"The HTTP 404 Not Found response status code indicates that the server cannot find the requested resource."